test/conformance/kuttl/validate/clusterpolicy/standard/enforce/not-allow-by-all-not-in/01-manifests.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: require-labels
+spec:
+  background: false
+  failurePolicy: Fail
+  rules:
+    - name: check-for-labels
+      match:
+        any:
+        - resources:
+            kinds:
+            - 'Deployment'
+      preconditions:
+        any:
+        - key: "{{ request.object.metadata.namespace }}"
+          operator: AllNotIn
+          value:
+            - kyverno
+      validate:
+        message: "label 'app.kubernetes.io/name' is required"
+        pattern:
+          metadata:
+            labels:
+              app.kubernetes.io/name: "?*"
+  validationFailureAction: Enforce
```

Comment:
Please use lower-case for the value so we can cherry-pick this to 1.8 branch.

Reply:
Ok. I will adjust ASAP.

Reply:
I'm sorry for the delay, because I was affected by COVID-19. I will update this pr today.